cfy_manager/components/restservice/restservice.py

```diff
@@ -306,10 +306,10 @@ def _validate_cluster_join(self):
 
         certs = config[SSL_INPUTS]
         internal_certs_provided = (
-            certs['internal_cert_path'] and certs['internal_key_path']
+                certs['internal_cert_path'] and certs['internal_key_path']
```

Comment:
This doesn't need an extra level of indentation

Reply:
Pycharm is a sneaky one, will remove